### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by created4GH**

### CODE REVIEW

File: src/components/UserCard/index.tsx [hunk 1]
```diff
@@ -13,36 +13,46 @@ interface Props {
...
+  const defaultInfo: string[] = [
     name.first + " " + name.last,
-    format(gender) + ", " + dob.age + " " + format("years old"),
-    format("Date of birth") + ": " + dob.date,
+    formatMessage({id: gender}) + ", " + dob.age + " " + formatMessage({id: "years old"}),
+    formatMessage({id: "Date of birth"}) + ": " + dob.date,
   ];
 
+  switch(isFullInfo){
+    case(true):
```
Comment (alt-a1r) [inline src/components/UserCard/index.tsx:33]: switch/case for boolean condition with two cases ?

File: src/components/UserCard/index.tsx [hunk 2]
```diff
@@ -13,36 +13,46 @@ interface Props {
...
+    const { email, phone, location, registered } = user!; 
+    userInfo = [...defaultInfo,
+      formatMessage({id: "Email"}) + ': ' + email,
+      formatMessage({id: "Phone"}) + ': ' + phone,
+      formatMessage({id: "Address"}) + ': ' + location.street.name + ", " + location.street.number,
```
Comment (alt-a1r) [inline src/components/UserCard/index.tsx:38]: Transform data to array and map it is not necessary. Use just layout to render this page

File: src/components/UserInfo/index.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ const UserInfo: React.FC = () => {
     if (selectedUser) {
         return (
             <div className="additional-info-container">
-                <AdditionalUserInfo selectedUser={selectedUser} />
+                <UserCard user={selectedUser} isFullInfo={true}/>
```
Comment (alt-a1r) [inline src/components/UserInfo/index.tsx:23]: Separated component was right way for this case

File: src/components/UserCard/index.tsx (hunk 1 above)
Comment (created4GitHub) [reply src/components/UserCard/index.tsx:33]: just to shown that i can use it)

File: src/components/UserInfo/index.tsx (hunk 3 above)
Comment (created4GitHub) [reply src/components/UserInfo/index.tsx:23]: I've thought that the logic of AdditionalUserInfo can be handled in UserCard according to your comment: "AdditionalUserInfo по сути лишний компонент. Это hoc в котором нет необходимости, его логику можно перенести в UserCard"

File: src/components/UserCard/index.tsx (hunk 2 above)
Comment (created4GitHub) [reply src/components/UserCard/index.tsx]: i've done it

**Pull request merged**